pkgs/by-name/es/eslint/package.nix

```diff
@@ -30,11 +30,16 @@ buildNpmPackage' rec {
     cp ${./package-lock.json} package-lock.json
   '';
 
-  npmDepsHash = "sha256-Nrcld0ONfjdSh/ItdbDMp6dXVFKoj83aaoGXDgoNE60=";
+  npmDepsHash = "sha256-F3EUANBvniczR7QxNfo1LlksYPxXt16uqJDFzN6u64Y=";
 
   dontNpmBuild = true;
   dontNpmPrune = true;
 
+  postFixup = ''
+    # Remove broken symlink
+    rm $out/lib/node_modules/eslint/node_modules/eslint-config-eslint
+  '';
+
```

Comment:
```suggestion
  postInstall = ''
    mkdir -p $out/lib/node_modules/eslint/packages
    cp -r $src/packages/eslint-config-eslint $out/lib/node_modules/eslint/packages
  '';
```
this?

Reply:
Don't know if we need to build it as a subproject and include it or can just copy it as-is